kubernetes/samples/scripts/create-rcu-schema/stop-db-service.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
```

Comment:
wrong year

Reply:
Modified all the scripts with following line
#!/bin/bash
# Copyright 2019, Oracle Corporation and/or its affiliates.  All rights reserved.
# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.